microsoftdefenderforendpoint_connector.py

```diff
@@ -3556,9 +3556,7 @@ def _check_date_format(self, action_result, date):
 
         try:
             time = datetime.strptime(date, DEFENDER_APP_DT_STR_FORMAT)
-            end_time = datetime.now(datetime.UTC)
-            if self._check_invalid_since_utc_time(time):
-                return action_result.set_status(phantom.APP_ERROR, LOG_UTC_SINCE_TIME_ERR)
+            end_time = datetime.utcnow()
```

Comment:
`datetime.utcnow()` should not be used because it's a naive datetime: https://blog.miguelgrinberg.com/post/it-s-time-for-a-change-datetime-utcnow-is-now-deprecated

```suggestion
end_time = datetime.now(datetime.UTC)
```

Reply:
Also, is there a reason we're removing the validation check? If the check is being removed here, should the `_check_invalid_since_utc_time()` function and `LOG_UTC_SINCE_TIME_ERR` constant be removed, or are they still used elsewhere?

Reply:
Updated 👍